ultraplot/axes/plot.py

```diff
@@ -3482,14 +3479,17 @@ def _apply_scatter(self, xs, ys, ss, cc, *, vert=True, **kwargs):
                 infer_rgb=infer_rgb,
                 **kw,
             )
+        # Move _parse_cycle before _parse_color
+        kw = self._parse_cycle(
+            xs.shape[1] if xs.ndim > 1 else 1, cycle_manually=cycle_manually, **kw
+        )
```

Comment:
The comment mentions moving the cycler parsing before the parsing of the color. However, this change undoes that. Why? Should we delete the comment or do we need to move this back?